packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -472,18 +486,28 @@ export class QuoteRequestor {
                 try {
                     if (typedMakerUrl.pairType === RfqPairType.Standard) {
                         const response = await this._quoteRequestorHttpClient.get(`${typedMakerUrl.url}/${quotePath}`, {
-                            headers: { '0x-api-key': options.apiKey },
+                            headers: { '0x-api-key': options.integrator.integratorId },
```

Comment:
Can we send this value twice, as the 0x-api-key header as well as 0x-integrator-id? That way we can one day remove the concept of 0x-api-key for quote served

Reply:
love it

Reply:
Mindblown. Thank you

Reply:
Just thinking about this a sec, @phil-ociraptor - if we create custom API keys that can only connect to a subset of services. Won't we risk doxxing integrators if there is some bug?

Example: we create a "Jump Trading" API key, which whitelists only Jump. A regression bug happens and suddently Wintermute can see requests from Jump Trading?